lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -138,9 +138,10 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
 
   /// The current list of windows,
   @override
-  Iterable<ui.FlutterView> get views => _windows.values;
-  Map<Object, ui.FlutterWindow> get windows => _windows;
-  final Map<Object, ui.FlutterWindow> _windows = <Object, ui.FlutterWindow>{};
+  Iterable<ui.FlutterView> get views => _views.values;
+  final Map<Object, ui.FlutterView> _views = <Object, ui.FlutterView>{};
+  ui.FlutterView? getViewById(Object id) => _views[id];
+  void addView(Object id, ui.FlutterView view) => _views[id] = view;
```

Comment:
Since this is all private, you could just replace all of this with something like:

```suggestion
  final Map<Object, ui.FlutterView> idToView = <Object, ui.FlutterView>{};
```

... and then elsewhere operate directly with that map.